src/styles/blocks/buttons.scss

```diff
@@ -0,0 +1,29 @@
+.buttons {
+  @include pageGrid;
+
+  &__config {
+    // grid-column: 1 / -1;
+    cursor: pointer;
+    background: $main-text-color;
+    padding-block: 14.5px;
+    border-radius: 8px;
+
+    color: #fff;
+    font-size: 16px;
+    line-height: 27px;
+    font-weight: 700;
+    cursor: pointer;
+    text-align: center;
+
+    &:hover {
+      background: #F8F8FA;
+      color: $main-text-color;
+      border: 2px solid #F0F1F4;
+    }
+
+    &:focus {
+      background: $main-text-color;
```

Comment:
The ':focus' state has the same styles as the default button state, consider adding different styles or removing this block if it's not needed.